src/app/models/evt-models.ts

```diff
@@ -637,7 +637,7 @@ export class CustodialHist extends GenericElement {
     structuredData: boolean;
     pEl?: Paragraph[];
```

Comment:
Is it possible to remove the optional field and assume that when not provided this field is a zero length array?

Reply:
I agree, but I think this optimization can be handled in another pr.